### PR TITLE
Don't re-export traits from crate root

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

**Comment by sdroege** [comment]
* gstreamer-rs MR: https://gitlab.freedesktop.org/gstreamer/gstreamer-rs/-/merge_requests/751
* gtk4-rs PR: https://github.com/gtk-rs/gtk4-rs/pull/323

File: gio/src/unix_socket_address.rs [hunk 1]
```diff
@@ -1,10 +1,10 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
+use crate::prelude::*;
 use crate::SocketAddress;
 use crate::UnixSocketAddress;
-use crate::UnixSocketAddressExt;
 use crate::UnixSocketAddressType;
-use glib::object::{Cast, IsA};
+use glib::prelude::*;
```
Comment (bilelmoussaoui) [inline gio/src/unix_socket_address.rs:7]: i believe gio prelude contains glib's prelude already no?

**Review by bilelmoussaoui — APPROVED** [review]

File: gio/src/unix_socket_address.rs (hunk 1 above)
Comment (sdroege) [reply gio/src/unix_socket_address.rs]: Indeed, removed here and in a few other places.

**Comment by bilelmoussaoui** [comment]
Something seems to be wrong with glib when generating the docs

**Comment by sdroege** [comment]
That passed before though... did we maybe get a new nightly in the last 3 hours and that broke things? :upside_down_face:

**Comment by sdroege** [comment]
Apparently just a case of cosmic rays.

**Pull request merged**